core/app/models/concerns/spree/multi_searchable.rb

```diff
@@ -0,0 +1,21 @@
+module Spree
+  module MultiSearchable
+    extend ActiveSupport::Concern
+
+    included do
+      def self.sanitize_query_for_multi_search(query)
+        ActiveRecord::Base.sanitize_sql_like(query.to_s.downcase.strip)
+      end
+
+      def self.multi_search_condition(model_class, attribute, query)
+        encrypted_attributes = model_class.encrypted_attributes.presence || []
+
+        if encrypted_attributes.include?(attribute.to_sym)
```

Comment:
This will improve the search for attributes that are not encrypted and will use `LIKE` in query